### PR TITLE
Ensure local scripts can be loaded

### PR DESCRIPTION
**Pull request by DonJayamanne**
Fixes https://github.com/microsoft/vscode-jupyter/issues/12843

### CODE REVIEW

File: src/webviews/webview-side/ipywidgets/dummy.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * We need to tell VS Code some way to register the Jupyter extension directory
+ * as a known root directory where from scripts can be loaded.
+ * If we do not do this and attempt to load scripts from the Jupyter extension directory,
+ * then VS Code will return a 401 error.
+ * One way to do this is to create a script file and define this as a pre-load script for the controller in the controller code.
+ * NOTE: Defining pre-load scripts in the package.json file does not work.
```
Comment (DonJayamanne) [inline src/webviews/webview-side/ipywidgets/dummy.ts:10]: Will file an issue and get this fixed within VS Code itself, this is a simple quick and dirty fix

File: src/notebooks/controllers/ipywidgets/message/commonMessageCoordinator.ts [hunk 2]
```diff
@@ -193,6 +204,9 @@ export class CommonMessageCoordinator {
         // In case the webview loaded earlier and it already sent the IPyWidgetMessages.IPyWidgets_Ready message
         // This way we don't make assumptions, we just query widgets and ask its its ready (avoids timing issues etc).
         webview.postMessage({ type: IPyWidgetMessages.IPyWidgets_IsReadyRequest, payload: undefined }).then(noop, noop);
+        // Send the IPyWidgets message immediately, sometimes
+        // the webview is ready before we get the request message.
+        sendIPyWidgetsVersion().catch(noop);
```
Comment (DonJayamanne) [inline src/notebooks/controllers/ipywidgets/message/commonMessageCoordinator.ts:209]: This change was to address a race condition

File: src/webviews/webview-side/ipywidgets/dummy.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/webviews/webview-side/ipywidgets/dummy.ts:10]: @roblourens /cc Here's the temporary & easy fix.
Will try to look into the issue in core as well (but this is an easy fix for now)

**Review by amunger — APPROVED** [review]

**Comment by DonJayamanne** [comment]
Damn it, looks like a bug in github actions, the smoke tests have run to completion, but the action has not detected it

**Pull request merged**